### PR TITLE
Api for channel

### PR DESCRIPTION
**Pull request by Sweex99**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review
- [x] no instance variables in partials
- [x] code is well formatted and has spaces!!!

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by Sweex99** [comment]
Screenshot from 2020-01-06 20-03-50

**Comment by Sweex99** [comment]
Screenshot from 2020-01-06 20-04-11

File: spec/controllers/api/v1/channels_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ChannelsController, type: :controller do
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:user, :admin) }
+
+  describe 'GET #index' do
+    it 'return organization by token' do
```
Comment (demchyna) [inline spec/controllers/api/v1/channels_controller_spec.rb]: навіщо тут let!(:admin) { create(:user, :admin) } ??? в цьому тесті?

File: spec/controllers/api/v1/organizations_controller_spec.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Api::V1::OrganizationsController, type: :controller do
   let!(:organization) { create(:organization) }
+  let!(:admin) { create(:user, :admin) }
```
Comment (demchyna) [inline spec/controllers/api/v1/organizations_controller_spec.rb]: і тут

File: spec/controllers/api/v1/channels_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      request.headers['token'] = organization.token
+      get :index
```
Comment (demchyna) [inline spec/controllers/api/v1/channels_controller_spec.rb]: може так працює?
get :index, headers: { 'token' => org.token }

**Pull request merged**